### ISSUE TITLE
Issue in demo-scene in unity.

### ISSUE THREAD

**Issue by Nelfstor** [melanchall/drywetmidi#245]
Hello! 

I've downloaded nativeless asset, run a demoScene and get an error:

DllNotFoundException: Melanchall_DryWetMidi_Native64 assembly:<unknown assembly> type:<unknown type> member:(null)
Melanchall.DryWetMidi.Multimedia.CommonApi64.Api_GetApiType () (at <1ee5422f8ce74a06831827bb5f773e0a>:0)
Melanchall.DryWetMidi.Multimedia.TickGeneratorSession.GetSessionHandle () (at <1ee5422f8ce74a06831827bb5f773e0a>:0)
Melanchall.DryWetMidi.Multimedia.HighPrecisionTickGenerator.EnsureSessionIsCreated () (at <1ee5422f8ce74a06831827bb5f773e0a>:0)
Melanchall.DryWetMidi.Multimedia.HighPrecisionTickGenerator.Start (System.TimeSpan interval) (at <1ee5422f8ce74a06831827bb5f773e0a>:0)
Melanchall.DryWetMidi.Multimedia.TickGenerator.TryStart (System.TimeSpan interval) (at <1ee5422f8ce74a06831827bb5f773e0a>:0)
Melanchall.DryWetMidi.Multimedia.MidiClock.Start () (at <1ee5422f8ce74a06831827bb5f773e0a>:0)
Melanchall.DryWetMidi.Multimedia.Playback.Start () (at <1ee5422f8ce74a06831827bb5f773e0a>:0)
DemoScript.StartPlayback () (at Assets/Melanchall/DryWetMIDI.Nativeless/Demo/DemoScript.cs:102)
DemoScript.Start () (at Assets/Melanchall/DryWetMIDI.Nativeless/Demo/DemoScript.cs:40)

I'm running Unity On Windows 10. And plan to build for Android. Could you tell me if i can fix this error.

**Comment by melanchall** [comment]
Hello,

Oh, my fault. I'll fix the script as soon as get back to my computer (unfortunately can't say when it will happen). But yes, you can fix the error. In the script you have this line:

```csharp
_playback = midiFile.GetPlayback(_outputDevice);
```

Change it to

```csharp
_playback = midiFile.GetPlayback(_outputDevice, new PlaybackSettings
{
    ClockSettings = new MidiClockSettings
    {
        CreateTickGeneratorCallback = () => new RegularPrecisionTickGenerator()
    }
});
```

You can set custom tick generator also as described here: [Tick generator](https://melanchall.github.io/drywetmidi/articles/playback/Tick-generator.html). In the article you can see that `RegularPrecisionTickGenerator` can be not a good choice. But it's a cross-platform one.

As for Android: if you plan to use only [core API](https://melanchall.github.io/drywetmidi/articles/dev/Supported-OS.html#core-api), that's OK. Can you please list functionality you want to use from the DryWetMIDI? What tasks do you plan to solve? Probably you don't want to use playback API at all.

**Comment by Nelfstor** [comment]
Thanks for your answer! 

For now i just need to read midi-files: note values and length, tempo, text from them. For project I'm working on now - i even don't need to play midi notes but i'll probably try to add this feature later. ( there should be some nativeless solutions in Unity for the cases when Midi General Synth can not be used).

**Comment by Nelfstor** [comment]
I've added your code and demo worked fine!

But then i get the same error when i tryied to use: OutputDevice.GetAll();

**Comment by melanchall** [comment]
Hmm, are you sure you use nativeless package?

**Comment by Nelfstor** [comment]
> Yes, i've checked that it is nativeless package.

Here is the link: 

[https://drive.google.com/file/d/1uTPcg1qrz4L9AyUiZX2m7ku7w6Z3ZKtK/view?usp=share_link](url)

**Comment by melanchall** [comment]
Fixed in 7.0.0 version.
